### ISSUE TITLE
docs: missing navigation in smaller screen display

### ISSUE THREAD

**Issue by womensrights** [cosmos/ibc-go#6502]
When using a smaller display you lose the navigation features in the header

<img width="896" alt="image" src="https://github.com/cosmos/ibc-go/assets/65018876/251736b5-36bb-4baa-92c2-f52ce4c05478">

compared to:

<img width="1091" alt="image" src="https://github.com/cosmos/ibc-go/assets/65018876/ace604f0-52de-43cc-8f56-a5a9e633ce55">

There should be a way to navigate to the ADRs, tutorials, github, light and dark mode toggle in the left navigation bar for mobile / smaller display users.

There is also the search and menu overlap problem which should be fixed in [#6501](https://github.com/cosmos/ibc-go/issues/6501)

**Comment by hungdinh82** [comment]
@womensrights does this issue still need to be resolved?

**Comment by womensrights** [comment]
yes this issue is still there, feel free to take it on but there will be a delay in review because of current priorities

**Comment by hungdinh82** [comment]
Do you have a specific design for this issue yet? I’d like to propose creating a BottomNavbar that includes sections for "Documentation", "Architecture Decision Records", "Tutorials", and Version.

**Comment by gjermundgaraba** [comment]
It is uncertain how we will deal with the docs site, so closing for now.
